cluster/addons/dns/kube2sky/README.md

```diff
@@ -10,16 +10,26 @@ containers.
 ## Namespaces
 
 Kubernetes namespaces become another level of the DNS hierarchy.  See the
-description of `-domain` below.
+description of `-domain` and `-pod_domain` below.
 
 ## Flags
 
-`-domain`: Set the domain under which all DNS names will be hosted.  For
-example, if this is set to `kubernetes.io`, then a service named "nifty" in the
+`-domain`: Set the domain under which all service DNS names will be hosted.  For
+example, if this is set to `svc.kubernetes.io`, then a service named "nifty" in the
 "default" namespace would be exposed through DNS as
-"nifty.default.kubernetes.io".
+"nifty.default.svc.kubernetes.io".
 
-`-verbose`: Log additional information.
+`-pod_domain`: Set the domain under which all DNS names for pods matching services
+will be hosted. If this is set to `pod.kubernetes.io`, then a pod named "dandy"
```

Comment:
This service is using two configurable subdomains - for Pods and Services. pod.something and svc.something. This is why I need two domains.

I rebased this PR last time long time ago. It might not be suitable for the current Kubernetes design. I'm still using this server in production. It works without any issues. If you think it's not a good fit for modern Kubernetes design, feel free to delete my PR.